packages/components-react/src/button/Button.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button as UtrechtButton, type ButtonProps as UtrechtButtonProps } from '@utrecht/button-react/dist/css'; //Utrecht/components-react/dist/css doesn't have css code included
```

Comment:
De button uit utrecht/components-react krijgt geen styling mee. (ik zal Robbert vragen of dit een issue is bij Utrecht)

Reply:
Lijkt opgelost in versie 7.1.0